### ISSUE TITLE
BatchNorm operator has multi output

### ISSUE THREAD

**Issue by SystemErrorWang** [onnx/onnx#3949]
# Bug Report

### Is the issue related to model conversion?
No

### Describe the bug
I am working on a pytorh model which contains Adaptive Instance Normalizaiton (https://github.com/NVlabs/MUNIT/blob/master/networks.py). When I convert it to onnx and use Netron to visualize it, I found each BatchNorm op has 5 outputs.
<img width="1072" alt="netron" src="https://user-images.githubusercontent.com/26077659/149772558-0e9cbf65-7d2f-4600-9a26-1e6fc51eb5be.png">
this problem happend in pytorch 1.10.1, while in pytorch 1.8.0, the batchnorm op behaves normal, but the onnxruntime results are different from the pytorch results.

### System information
- OS Platform and Distribution: MacOS 10.15.7
- ONNX version (*e.g. 1.7*):  1.10.2
- Python version: 3.6.10
- GCC/Compiler version (if compiling from source):
- CMake version:
- Protobuf version:
- Visual Studio version (if applicable):

### Reproduction instructions
the code to export onnx are listed below:
```
image1 = torch.randn(1, 3, 224, 192)
image2 = torch.randn(1, 3, 112, 112)
model = FullModel(num_ch=28)

weight = torch.load('model.pth', map_location='cpu')
model.generator.load_state_dict(weight['g'])
#model.generator.eval()
model.encoder.load_state_dict(weight['e'])
#model.encoder.eval()
model.eval()
with torch.no_grad():
    torch.onnx.export(model, (image1, image2), 'test.onnx', opset_version=11)
```

**Comment by jcwchen** [comment]
Hi @SystemErrorWang,
So it seems that `torch.onnx.export` behaves differently between different PyTorch versions but with the same specified opset_version? If so, it should be a question for the converter (like why the conversion behaves differently). I would suggest you open an issue under PyTorch repo to get the best help from the converter team.

Close here now. Feel free to reopen it if you still have other ONNX questions. Thanks!
